src/Enghouse/readme.md

```diff
@@ -0,0 +1,100 @@
+# Enghouse Interactive CCaaS Integration Example
+
+The **Enghouse CCaaS** integration enables integration between IBM Watson Assistant (WA) and Enghouse Interactive (EI) **CCaaS** tenants' live agents.
+
+## Overview
+
+The integration attempts to make it as simple as possible for Watson Assistance users with EI **CCaaS** live agents.
+
+### Architecture
+
+The EI **CCaaS**  integration consists of two main components: hosting web page and backend Contact Center platform . Below is client side componenet Architecture:
+
+![EI **CCaaS** Watson Asistance Integration Architecture Overview](ibmWatson.jpg)
+
+The client-side component manages the communication between the user and the agent. It implements the service desk API that is fully supported by the Watson Assistant web chat integration. (For more information about this API, see [ServiceDesk API](https://github.com/watson-developer-cloud/assistant-web-chat-service-desk-starter/blob/main/docs/API.md)).
+
+The communication uses the EI CustomChat widget component.  This can be referenced using the EI **CCaaS** default page or the [hosting page](./client/indexWatson.html) can set the `watsonAssistantChatOptions` prior to loading the EnghouseCCaaSWatsonAssistant.js for more advanced features, e.g.:
+
+```
+<script>
```

Comment:
I made a few suggestions in your example HTML file. I would just copy those here.